classes/Category.php

```diff
@@ -2043,6 +2043,23 @@ public function getProductsWs()
 		ORDER BY `position` ASC');
     }
 
+    /**
+     * Get products of the category that will be deleted
+     */
+    public function getProductsBeforeDeletion(): array
```

Comment:
I think the name of this function can be a bit more generic, like `getAllCategoryProducts()`

Reply:
what about `getCategoryProducts`? 

Thanks for CR

Reply:
@arouiadib Ok for this naming